### PR TITLE
add certificates to ci.yml

### LINKED ISSUE #2896: Archboard review

### PR DESCRIPTION
**Pull request by gearama**
fixes #2896 
# Pull Request Checklist

Please leverage this checklist as a reminder to address commonly occurring feedback when submitting a pull request to make sure your PR can be reviewed quickly:

See the detailed list in the [contributing guide](https://github.com/Azure/azure-sdk-for-cpp/blob/main/CONTRIBUTING.md#pull-requests).

- [x] [C++ Guidelines](https://azure.github.io/azure-sdk/cpp_introduction.html)
- [x] Doxygen docs
- [x] Unit tests
- [x] No unwanted commits/changes
- [x] Descriptive title/description
  - [x] PR is single purpose
  - [x] Related issue listed
- [x] Comments in source
- [x] No typos
- [x] Update changelog
- [x] Not work-in-progress
- [x] External references or docs updated
- [x] Self review of PR done
- [x] Any breaking changes?

### CODE REVIEW

**Review by vhvb1989 — APPROVED** [review]
for the night is dark!

File: sdk/keyvault/ci.yml [hunk 1]
```diff
@@ -44,3 +44,6 @@ stages:
         - Name: azure-security-keyvault-secrets
           Path: azure-security-keyvault-secrets
           VcpkgPortName: azure-security-keyvault-secrets-cpp
+        - Name: azure-security-keyvault-certificates
+          Path: azure-security-keyvault-certificates
+          VcpkgPortName: azure-security-keyvault-certificates-cpp
```
Comment (ahsonkhan) [inline sdk/keyvault/ci.yml:49]: Please do verify that the vcpkg daily PR has the runs showing up.

And that it is green. Doesn't look like certificates are showing up atm.

https://github.com/microsoft/vcpkg/pull/17119

@vhvb1989 it looks like vcpkg CI is failing (with keys/secrets in there).

https://dev.azure.com/vcpkg/public/_build/results?buildId=62285&view=logs&j=878666d5-db33-5b27-9e7d-b0c7ee352005&t=d914c609-a9aa-5880-221f-cbd46db9069e&l=2177

```text
2021-11-01T23:21:25.2251450Z Error: Building package azure-security-keyvault-keys-cpp:x86-windows failed with: BUILD_FAILED
2021-11-01T23:21:25.2256758Z Elapsed time for package azure-security-keyvault-keys-cpp:x86-windows: 4.357 s
2021-11-01T23:21:25.2262087Z Starting package 10/16: azure-security-keyvault-secrets-cpp:x86-windows
2021-11-01T23:21:25.2267420Z Building package azure-security-keyvault-secrets-cpp[core]:x86-windows...
2021-11-01T23:21:28.5805852Z -- Using cached Azure-azure-sdk-for-cpp-041feefd67de7d25d82f2b61b826e8ff86ba8c63.tar.gz.
2021-11-01T23:21:28.5811305Z -- Extracting source D:/downloads/Azure-azure-sdk-for-cpp-041feefd67de7d25d82f2b61b826e8ff86ba8c63.tar.gz
2021-11-01T23:21:28.5817053Z -- Using source at D:/buildtrees/azure-security-keyvault-secrets-cpp/src/ff86ba8c63-e23051d11c.clean
2021-11-01T23:21:28.5822757Z -- Configuring x86-windows
2021-11-01T23:21:28.5828533Z -- Building x86-windows-dbg
2021-11-01T23:21:28.5834135Z CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:155 (message):
2021-11-01T23:21:28.5839669Z     Command failed: D:/downloads/tools/cmake-3.21.1-windows/cmake-3.21.1-windows-i386/bin/cmake.exe --build . --config Debug --target install -- -v -j33
2021-11-01T23:21:28.5845872Z     Working Directory: D:/buildtrees/azure-security-keyvault-secrets-cpp/x86-windows-dbg
2021-11-01T23:21:28.5851129Z     See logs for more information:
2021-11-01T23:21:28.5856990Z       D:\buildtrees\azure-security-keyvault-secrets-cpp\install-x86-windows-dbg-out.log
2021-11-01T23:21:28.5859147Z 
2021-11-01T23:21:28.5864942Z Call Stack (most recent call first):
2021-11-01T23:21:28.5870751Z   D:/installed/x64-windows/share/vcpkg-cmake/vcpkg_cmake_build.cmake:102 (vcpkg_execute_build_process)
2021-11-01T23:21:28.5876135Z   D:/installed/x64-windows/share/vcpkg-cmake/vcpkg_cmake_install.cmake:41 (vcpkg_cmake_build)
2021-11-01T23:21:28.5881613Z   ports/azure-security-keyvault-secrets-cpp/portfile.cmake:14 (vcpkg_cmake_install)
2021-11-01T23:21:28.5886837Z   scripts/ports.cmake:142 (include)
```

cc @antkmsft, @danieljurek if anything pops out to you :)

Comment (ahsonkhan) [reply sdk/keyvault/ci.yml:49]: The debug out build log tell us what went wrong, @vhvb1989:

```text
D:\buildtrees\azure-security-keyvault-keys-cpp\src\ff86ba8c63-e23051d11c.clean\sdk\keyvault\azure-security-keyvault-keys\src\key_client.cpp(9): fatal error C1083: Cannot open include file: 'azure/keyvault/shared/keyvault_shared.hpp': No such file or directory
```

And
```text
[5/33] C:\PROGRA~2\MICROS~3\2019\ENTERP~1\VC\Tools\MSVC\1429~1.301\bin\Hostx64\x64\cl.exe   /TP -DAZ_CORE_DLL -DAZ_RTTI -DAZ_SECURITY_KEYVAULT_KEYS_BEING_BUILT -DAZ_SECURITY_KEYVAULT_KEYS_DLL -DBUILD_TRANSPORT_WINHTTP_ADAPTER -Dazure_security_keyvault_keys_EXPORTS -ID:\buildtrees\azure-security-keyvault-keys-cpp\src\ff86ba8c63-e23051d11c.clean\sdk\keyvault\azure-security-keyvault-keys\inc -ID:\buildtrees\azure-security-keyvault-keys-cpp\src\ff86ba8c63-e23051d11c.clean\sdk\keyvault\azure-security-keyvault-keys\sdk\keyvault\azure-security-keyvault-shared\inc -ID:\installed\x64-windows\include /nologo /DWIN32 /D_WINDOWS /W4 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  /permissive- /W4 /wd5031 /wd4668 /wd4820 /wd4255 /wd4710 -std:c++14 /showIncludes /FoCMakeFiles\azure-security-keyvault-keys.dir\src\cryptography\cryptography_client.cpp.obj /FdCMakeFiles\azure-security-keyvault-keys.dir\ /FS -c D:\buildtrees\azure-security-keyvault-keys-cpp\src\ff86ba8c63-e23051d11c.clean\sdk\keyvault\azure-security-keyvault-keys\src\cryptography\cryptography_client.cpp
FAILED: CMakeFiles/azure-security-keyvault-keys.dir/src/cryptography/cryptography_client.cpp.obj 
C:\PROGRA~2\MICROS~3\2019\ENTERP~1\VC\Tools\MSVC\1429~1.301\bin\Hostx64\x64\cl.exe   /TP -DAZ_CORE_DLL -DAZ_RTTI -DAZ_SECURITY_KEYVAULT_KEYS_BEING_BUILT -DAZ_SECURITY_KEYVAULT_KEYS_DLL -DBUILD_TRANSPORT_WINHTTP_ADAPTER -Dazure_security_keyvault_keys_EXPORTS -ID:\buildtrees\azure-security-keyvault-keys-cpp\src\ff86ba8c63-e23051d11c.clean\sdk\keyvault\azure-security-keyvault-keys\inc -ID:\buildtrees\azure-security-keyvault-keys-cpp\src\ff86ba8c63-e23051d11c.clean\sdk\keyvault\azure-security-keyvault-keys\sdk\keyvault\azure-security-keyvault-shared\inc -ID:\installed\x64-windows\include /nologo /DWIN32 /D_WINDOWS /W4 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  /permissive- /W4 /wd5031 /wd4668 /wd4820 /wd4255 /wd4710 -std:c++14 /showIncludes /FoCMakeFiles\azure-security-keyvault-keys.dir\src\cryptography\cryptography_client.cpp.obj /FdCMakeFiles\azure-security-keyvault-keys.dir\ /FS -c D:\buildtrees\azure-security-keyvault-keys-cpp\src\ff86ba8c63-e23051d11c.clean\sdk\keyvault\azure-security-keyvault-keys\src\cryptography\cryptography_client.cpp
D:\buildtrees\azure-security-keyvault-keys-cpp\src\ff86ba8c63-e23051d11c.clean\sdk\keyvault\azure-security-keyvault-keys\src\cryptography\cryptography_client.cpp(9): fatal error C1083: Cannot open include file: 'azure/keyvault/shared/keyvault_shared.hpp': No such file or directory
```

Comment (vhvb1989) [reply sdk/keyvault/ci.yml:49]: Fix for the VCPKG build: https://github.com/Azure/azure-sdk-for-cpp/pull/3034

**Pull request merged**